setup.py

```diff
@@ -40,6 +40,7 @@
         'pylint==1.6.5',
         'six>=1.10.0',
         'typing>=3.5.3.0',
+        'GitPython==2.1.1',
```

Comment:
If `shopify_python.git_utils` is part of our public API, then `GitPython` remains here. Likewise with `pylint`.